lib/vrl/stdlib/Cargo.toml

```diff
@@ -66,6 +66,7 @@ default = [
     "encode_percent",
     "ends_with",
     "exists",
+    "find_table_row",
```

Comment:
I realize our [design document](https://github.com/timberio/vector/blob/jean/vrl-design-doc/lib/vrl/DESIGN.md#naming) is still being iterated on, but it mentions:

> Favor explicit verbose names over terse ones (e.g. `parse_aws_cloudwatch_log_subscription_message` over `parse_aws_cwl_msg`).

Given that, in addition to the fact that we already use `get_*` in at least one place (`get_env_var`) with similar semantics (return one value, or error if none is found), I suggest:

`get_enrichment_table_record`

Reply:
Marking this as "unresolved", so that we don't lose track of this (feel free to mark it as resolved if we're tracking this in an issue somewhere).

Reply:
Ah wait, actually you _did_ change the name, but it appears you didn't change the feature name, probably an oversight?